roles/engineblock/tasks/main.yml

```diff
@@ -162,30 +162,13 @@
   when:
     - engine_crt_not_in_inventory | bool
 
-- name: Configure EngineBlock
-  template:
-    src: "{{ item }}.j2"
-    dest: "/etc/openconext/{{ item }}"
-  with_items:
-    - engineblock.ini
-  notify:
-    - "restart php72-fpm"
-
 - name: Make the SYMFONY_ENV var available for scripts
   become: yes
   lineinfile:
     dest: "/etc/environment"
     state: present
     line: "SYMFONY_ENV={{ engine_apache_symfony_environment }}"
 
-# This should always run before any scripts run, it ensures that the env specific
-# configuration is loaded correctly.
-- name: Prepare EngineBlock environment
-  shell: php72 bin/composer/dump-required-ini-params.sh
```

Comment:
Here we used a composer we apparently already have available. So why is it necessary to install it in this PR?

Reply:
I don't get it, `/bin/composer/` seems to be a folder here?

I've tested the changes in EB with the install from branch task. That task uses the `makeRelease.sh` script in EB and to run that script Composer is needed. I saw Composer was installed earlier on, but I saw efforts of Bart to remove Composer from Deploy and as far as I could see it should only be used when building a tar. I'ver asked Bart why this change was needed and he stated that he didn't like Composer in production and that's something I would totally agree on. So I decided to only added it to the 'install from branch' task were it's needed to build an artifact from scratch and seems to be there to test EB in deploy.

https://github.com/OpenConext/OpenConext-deploy/commit/4f0ef664af88151449e8ecbc9823f92f203d76a2

Reply:
My bad, indeed there's no composer now. Might make sense to move this to the php role in the future if more apps would work in a similar way. But OK for now.